src/util/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -111,6 +111,7 @@ describe("logSingleWord", () => {
 
 describe("CollectAllInvalidWordsInIssues", () => {
 	it("Creates a correct Issue-array for one wrong word containing only this word", () => {
+		emptyIssueList();
```

Comment:
You can use the beforeEach function with these. If it is being used in a describe block it will only be called before each tests inside the describe block.
https://jestjs.io/docs/en/setup-teardown.html

Reply:
Good suggestion! I added the beforeEach now. However, the approval got stale because of that. Could you renew it again?

Reply:
Sure thing.